backends/apple/coreml/test/test_coreml_partitioner.py

```diff
@@ -68,15 +68,23 @@ def test_vit_skip_conv(self):
             )
         )
 
+        conv_block = ["aten.convolution.default", "executorch_call_delegate"]
+        safe_softmax_block = [
+            "getitem",
+            "getitem",
+            "getitem",
+            "getitem",
+            "aten.any.dim",
+            "executorch_call_delegate",
+        ]
```

Comment:
In the image @shoumikhin showed me
![image](https://github.com/user-attachments/assets/5232e5e6-5001-4024-ba80-1d9fbd566119)
I thought each safe softmax block should be
```
"executorch_call_delegate",
"getitem",
"aten.any.dim",
"executorch_call_delegate",
"getitem",
```
?

Reply:
So the script that generated this https://gist.github.com/drisspg/b4087fc8aad510926c2cbe24e89f8e3b since I was having trouble building the actual PR with executorch: https://github.com/pytorch/pytorch/pull/131863

Reply:
Make sense 👍 then as long as CI green I believe this is good